NDTensors/src/lib/Sectors/src/category_product.jl

```diff
@@ -68,44 +245,42 @@ function CategoryProduct(pairs::Pair...)
   return CategoryProduct(NamedTuple{keys}(vals))
 end
 
-function categories_fusion_rule(A::NamedTuple, B::NamedTuple)
-  qs = [A]
-  for (la, lb) in zip(pairs(intersect_keys(A, B)), pairs(intersect_keys(B, A)))
-    @assert la[1] == lb[1]
-    fused_vals = ⊗(la[2], lb[2])
-    qs = [union_keys((; la[1] => v), q) for v in fused_vals for q in qs]
-  end
-  # Include sectors of B not in A
-  qs = [union_keys(q, B) for q in qs]
-  return qs
+function SymmetryStyle(NT::Type{<:NamedTuple})
```

Comment:
Same comment as above for the `Tuple` version, I think it would be better to rename this function to `categories_symmetrystyle` to follow the convention that we call functions `categories_*(...)` that act on the storage of a `CategoryProduct`.